vllm/engine/arg_utils.py

```diff
@@ -1295,16 +1295,9 @@ def _is_v1_supported_oracle(self, model_config: ModelConfig) -> bool:
 
         # No XFormers so far.
         V1_BACKENDS = [
-            "FLASH_ATTN_VLLM_V1",
-            "FLASH_ATTN",
-            "PALLAS",
-            "PALLAS_VLLM_V1",
-            "TRITON_ATTN_VLLM_V1",
-            "TRITON_MLA",
-            "FLASHMLA",
-            "FLASHINFER",
-            "FLASHINFER_VLLM_V1",
-            "ROCM_AITER_MLA",
+            "FLASH_ATTN_VLLM_V1", "FLASH_ATTN", "PALLAS", "PALLAS_VLLM_V1",
+            "TRITON_ATTN_VLLM_V1", "TRITON_MLA", "FLASHMLA", "FLASHINFER",
+            "FLASHINFER_VLLM_V1", "ROCM_AITER_MLA", "FLEX_ATTENTION_VLLM_V1"
```

Comment:
nit: please append `,` at the end to minimize the code diff.
```suggestion
            "FLASH_ATTN_VLLM_V1", "FLASH_ATTN", "PALLAS", "PALLAS_VLLM_V1",
            "TRITON_ATTN_VLLM_V1", "TRITON_MLA", "FLASHMLA", "FLASHINFER",
            "FLASHINFER_VLLM_V1", "ROCM_AITER_MLA", "FLEX_ATTENTION_VLLM_V1",
```